examples/01_plotting/plot_surf_stat_map.py

```diff
@@ -0,0 +1,58 @@
+"""
+Demonstrate basic surface plotting with plot_surf_stat_map
+==========================================================
```

Comment:
May be you can write few lines about what this example does. For example, I don't know what ```nki``` is ?
stating that we are fetching so and so datasets and visualizing them.